schema/partner_shows.js

```diff
@@ -0,0 +1,40 @@
+import gravity from '../lib/loaders/gravity';
+import PartnerShow from './partner_show';
+import {
+  GraphQLString,
+  GraphQLList,
+  GraphQLInt,
+  GraphQLBoolean
+} from 'graphql'
+
+let PartnerShows = {
+  type: new GraphQLList(PartnerShow.type),
+  description: 'A list of PartnerShows',
+  args: {
+    size: {
+      type: GraphQLInt
+    },
+    sort: {
+      type: GraphQLString
+    },
+    displayable: {
+      type: GraphQLBoolean
```

Comment:
Yea, I guess it's easier to add stuff if we need, but it's harder to remove things that other stuff may have started to depend on, I will remove :+1: